packages/framework/azure-client/.eslintrc.js

```diff
@@ -10,4 +10,8 @@ module.exports = {
     "parserOptions": {
         "project": ["./tsconfig.json", "./src/test/tsconfig.json"]
     },
+    "rules": {
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "no-null/no-null": "off",
```

Comment:
We should special case specific lines instead of blanket adding it to the package rule set if we can.